test/ginkgo/resize_test.go

```diff
@@ -0,0 +1,113 @@
+package test_test
+
+import (
+	//"bytes"
+	//"encoding/json"
+	//"fmt"
+	//"io"
+	//"os"
+	//"os/exec"
+	//"os/user"
+	//"path/filepath"
+	"runtime"
+	//"strings"
+	//"syscall"
+	//"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	//"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("start flags", func() {
+
+	Describe("memory, cpus, disk", func() {
+
+		It("setup CRC", func() {
+			Expect(RunCRCExpectSuccess("setup")).To(ContainSubstring("Setup is complete"))
+		})
+
+		It("start CRC with custom memory, cpu, and disk", func() {
+			Expect(RunCRCExpectSuccess("start", "-b", bundleLocation, "-p", pullSecretLocation, "--memory", "12000", "--cpus", "6", "--disk-size", "40")).To(ContainSubstring("Started the OpenShift cluster"))
+		})
+
+		switch os := runtime.GOOS; os {
+		case "linux":
+
+			It("check memory size", func() {
+				out, err := RunOnHost("ssh", "-i", credPath, "core@192.168.130.11", "cat", "/proc/meminfo")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(out).Should(MatchRegexp(`MemTotal:[\s]*11\d{6}`))
+			})
+
+			It("check number of cpus", func() {
+				out, err := RunOnHost("ssh", "-i", credPath, "core@192.168.130.11", "cat", "/proc/cpuinfo")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(out).Should(MatchRegexp(`processor[\s]*\:[\s]*5`))
+			})
+			It("check size of VM disk", func() {
+				out, err := RunOnHostWithPrivilege("virsh", "vol-info", "crc.qcow2", "crc")
```

Comment:
Check the size inside the VM so that it will  be the same check for all platforms.